src/styles/blocks/card.scss

```diff
@@ -0,0 +1,120 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background-color: $white-color;
+  border: 1px solid $gray-elements-color;
+  border-radius: 5px;
+
+  @include content-inline-padding;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 32px 0 40px;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    background-position: center;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+    margin-bottom: 16px;
+  }
+
+  &__title {
+    text-decoration: none;
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__title:hover {
+    color: $blue-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-count {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $main-color;
+    display: block;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: $secondary-color;
+  }
+
+  &__dollar {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__buy {
+    box-sizing: border-box;
+    width: 100%;
+    height: 40px;
+    margin-bottom: 16px;
```

Comment:
button shouldn't have margin (if you use margin-bottom for other elements above)